grow/pods/catalogs.py

```diff
@@ -118,13 +122,18 @@ def init(self, template_path, include_header=False):
         self.fuzzy = False
         self.save(include_header=include_header)
 
-    def update_using_catalog(self, catalog_to_merge, use_fuzzy_matching=False):
+    def update_using_catalog(self, catalog_to_merge, use_fuzzy_matching=False,
+                             include_obsolete=False):
         super(Catalog, self).update(
             catalog_to_merge, no_fuzzy_matching=(not use_fuzzy_matching))
+        # Don't use gettext's obsolete functionality as it polutes files: merge
```

Comment:
Oh, maybe. I see what you've done, but I don't know how the default functionality pollutes files other than leaving the terms in. What does it do exactly that's bad? I think obsolete terms in PO files are just comments preceded with `#~`. IMO if we could support that default obsolete behavior/syntax (when `--include-obsolete` is used) that'd be good? Feel free to make that outside the scope of this PR though.